module4/assignment4.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>First Form</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+  <link rel="stylesheet" href="_css/themes/ctheme1.css">
+  <link rel="stylesheet" href="_css/themes/jquery.mobile.icons.min.css">
+
+
+</head>
+
+<body>
+  <!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/4/2019
+-->
+  <div data-role="header">
+    <a data-ajax="false" href="../index.html" data-icon="home" rel="external">Home</a>
+    <h1>Ben Cook</h1>
+  </div>
+
+  <div data-role="content">
+    <form action="http://lolberding.portfolios.mccinfo.net/info_survey/bistrosurvey_insert.php" method="post">
```

Comment:
This form action is incorrect. Looks like you were referencing the old pdf of a previous class' assignment. My assignments are all in Google docs. The embedded instructions (and now the linked instructions) have the correct php script URL. You'll want to get this corrected for the challenge - as I grade you on being able to verify a write to the db. Let me know if you have questions.